helper_test.go

```diff
@@ -972,3 +972,28 @@ func randomString(t *testing.T) string {
 	}
 	return v
 }
+
+// skips a test if ENABLE_TFE is not set to equal "1"
+// this is used to conditionally skip tests if not running against a Terraform
+// Enterprise instance. Effectively these tests will only run against a Terraform
+// Enterprise instance.
+func skipIfNotEnterprise(t *testing.T) {
+	if !enterpriseEnabled() {
+		t.Skip("Skipping Terraform Enterprise related test. Set ENABLE_TFE=1 to run.")
```

Comment:
Nit: For this and the next one, maybe add some mention of 'only'. A test applicable to both platforms is related to TFE, so this may be too vague for someone seeing this the first time. "Skipping test only applicable to Terraform Enterprise." or similar.